### PR TITLE
fix(#2000): relative prop date picker React wrapper

### PR DESCRIPTION
**Pull request by syedszeeshan**
# Before (the change)

# After (the change)

## Make sure that you've checked the boxes below before you submit the PR

- [x] I have read and followed the [setup steps](https://goa-dio.atlassian.net/wiki/spaces/DS/pages/3086385489/Contribution+-+web+content#%5BinlineExtension%5DCode-contributions)
- [x] I have created necessary unit tests
- [ ] I have tested the functionality in both React and Angular.

### CODE REVIEW

**Comment by lizhuomeng71** [comment]
This issue is verified for angular and react. relative property is added for react. but seems that attribute is no longer needed.

File: libs/react-components/src/lib/date-picker/date-picker.tsx [hunk 1]
```diff
@@ -8,6 +8,7 @@ interface WCProps extends Margins {
   error?: boolean;
   min?: string;
   max?: string;
+  relative?: boolean;
```
[the remark this answers is not in the document]
Comment (syedszeeshan) [reply libs/react-components/src/lib/date-picker/date-picker.tsx:11]: Yes, once this PR is approved/merged, I am going to crearte a PR to update docs

**Review by chrisolsen — DISMISSED** [review]

**Review by chrisolsen — APPROVED** [review]

**Pull request merged**